bits/sendgrid/sendgrid.go

```diff
@@ -83,6 +84,15 @@ var _ = pubsub.NewSubscription(
 func sendEmail(ctx context.Context, event *EmailPreparedEvent) error {
 	// Creating an email
 	message := mail.NewSingleEmail(&event.From, event.Subject, &event.To, event.PlainTextContent, event.HTMLContent)
+	// Skipping sending an email in a non-production environment
+	if encore.Meta().Environment.Type != encore.EnvProduction {
+		rlog.Info(
+			"email sending was skipped in a non-production environment.",
+			"env:", encore.Meta().Environment.Type,
+			"message:", message)
```

Comment:
The keys shouldn't contain colons (just `"env"` and `"message"`)

Reply:
Fixed 
b76a820c30d390aea1a3f04fca74168f263ed0a7